app/v1/views/users.py

```diff
@@ -0,0 +1,110 @@
+from flasgger import swag_from
+from flask import jsonify, request, make_response, Blueprint
+import jwt
+import datetime
+
+from app.v1.models.users import DbUsers
+from app.v1.models.caterers import DbCaterers
+
+SECRET_KEY ='secretKey4512yek'
```

Comment:
Why do you have this here? Never hard-code secret_keys or passwords they should be put in the env file and never pushed to github